.moban.dt/setup.py.jj2

```diff
@@ -0,0 +1,11 @@
+{% extends 'coala-setup.py.jj2' %}
+
+{% block custom_requirements %}
```

Comment:
Even `setup.py.jj2` can be removed now. Since we are using the basic coala template.